docs/tx-malleability.md

```diff
@@ -0,0 +1,68 @@
+# Cosmos SDK Transaction Malleability Risks
```

Comment:
A gyst I am getting from this doc is:

1. using `SIGN_MODE_DIRECT` poses little/no risks
2. all of the risks outlined definitely apply to `SIGN_MODE_LEGACY_AMINO`

Is my understanding correct?  If so, I think we should be very explicit about that in this doc.  On top of that, we should illustrate where and why clients use the different sign modes

Reply:
That's basically accurate, `SIGN_MODE_DIRECT` only exposes signature malleability risks which no sign mode can control for.

I can add a section on sign modes plus some recommendations. A simple improvement to amino JSON would be including TxBody + AuthInfo hashes like textual does. That would address most of the issues.